pkg/config/config_template.yaml

```diff
@@ -146,6 +146,11 @@ api_key:
 #
 # The port for the go_expvar server
 # dogstatsd_stats_port: 5000
+#
+# If you want all statsd metrics coming from this host to be namespaced
```

Comment:
I think we could improve the wording from agent5 with something like:
```
# If you want all DogStatsd metrics coming from this host to be namespaced
# you can configure a namespace below. Each metric received will be prefixed
# with that namespace before it's sent to Datadog.
# statsd_metric_namespace:
```

What do you think ?

Reply:
Sounds good to me!